docling_core/types/doc/document.py

```diff
@@ -677,6 +678,89 @@ class PictureItem(FloatingItem):
 
     annotations: List[PictureDataType] = []
 
+    # Convert the image to Base64
+    def _image_to_base64(self, pil_image, format="PNG"):
+        buffered = BytesIO()
+        pil_image.save(buffered, format=format)  # Save the image to the byte stream
+        img_bytes = buffered.getvalue()  # Get the byte data
+        img_base64 = base64.b64encode(img_bytes).decode(
+            "utf-8"
+        )  # Encode to Base64 and decode to string
+        return img_base64
+
+    def export_to_markdown(
+        self,
+        doc: "DoclingDocument",
+        add_caption: bool = True,
+        image_placeholder: str = "<!-- image -->",
+        image_mode: ImageRefMode = ImageRefMode.EMBEDDED,
+        filename: Optional[Path] = None,
```

Comment:
Putting a filename argument here as optional argument is not the way export methods should work. Current export methods only return an in-memory representation and do _not_ write to file-system. See other comment on `export_to_dict`.